website/docs/providers/future_provider/config_consumer/cgen_config_consumer.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
```

Comment:
For the same of easy copy-paste, could could name the files in those folders:

```
codegen.dart
hooks_codegen.dart
raw.dart
hooks.dart
index.tsx
```
?


Reusing the same names allows cloning the folder without having to modify the `index.tsx` everytime.

Reply:
Oh, I will work on it now